frontend/simple/model/contracts/mailbox.js

```diff
@@ -1,11 +1,18 @@
 'use strict'
 
 import {DefineContract} from '../utils.js'
+import {
+  objectOf,
+  arrayOf,
+  string,
+  object,
+  optional
+} from 'flow-typer-js'
 
 export default DefineContract({
   'MailboxContract': {
     isConstructor: true,
-    validate: function (data) {},
+    validate: object,
```

Comment:
Don't we need to specify the object properties?

Reply:
These aren't defined / don't exist yet.